README.md

```diff
@@ -1,9 +1,38 @@
-# webmock-proxy
+# Webmock-proxy - Record external API interactions, Provide fast and safe testing.
+Webmock-proxy is "Man in the middle" proxy -- allow you to external API test without HTTP/S connection. Has two functions "Record" and "Mock".
 
-## Developement
+## Installation
 
 ```bash
+$ git clone git@github.com:wantedly/webmock-proxy.git
+$ cd webmock-proxy
 $ make deps
```

Comment:
This is not about README but this make is not common flow.

why don't you include `make deps` part into `make`?